std/conv.d

```diff
@@ -4075,7 +4077,7 @@ $(D T) is $(D @safe).
 Returns: A pointer to the newly constructed object.
  */
 T emplace(T, Args...)(void[] chunk, auto ref Args args)
-    if (is(T == class))
+    if (is(T == class) && !isAbstractClass!T)
```

Comment:
Can you turn this into a `static assert()`? A template constraint only makes sense if we'd want to add an overload for abstract classes, plus static assert gives a nicer error message.